app/javascript/views/statements/Statements.tsx

```diff
@@ -65,6 +71,49 @@ export const DisplayPaymentDate = (paymentDate: string) => {
   return "--";
 };
 
+export const GetParameterCaseInsensitive = (object, key) => {
+  return object[
+    Object.keys(object).find((k) => k.toLowerCase() === key.toLowerCase())
+  ];
+};
+
+export const DepositBreakdown = (props) => {
+  const id = props.name + "-" + Math.random();
```

Comment:
If there's an identical id, nothing should happen right?

Reply:
Correct, there would only be 2 things hovering over each other, so not that serious of a problem just a visual bug that `Math.random()` helps assist with